packages/devextreme/js/core/renderer.d.ts

```diff
@@ -36,6 +36,14 @@ export interface dxElementWrapper {
 
   detach(): this;
 
+  /**
+   * Iterates over each element in the set of matched elements, executing the provided function for each element.
+   *
+   * @param {(this: Element, index: number, element: Element) => boolean} func - A function to execute for each matched element.
+   *        The function receives the index position of the element in the set and the element itself as arguments.
+   *        If the function returns `false`, the iteration will stop.
+   * @returns {this} The current instance for chaining.
+   */
```

Comment:
Let's not add doc-comments in public .d.ts files. They might interfere with metadata generations

Reply:
removed